From 4c95dc87165a439698f63c3861c7c17b1fe2193a Mon Sep 17 00:00:00 2001
From: Bernard Cafarelli <bernard.cafarelli@gmail.com>
Date: Mon, 7 Sep 2020 14:29:22 +0200
Subject: [PATCH] Fix data testsuite path

Add separator between base path and set path if base path is set

Fixes #299
---
 src/nvtt/tests/testsuite.cpp | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/nvtt/tests/testsuite.cpp b/src/nvtt/tests/testsuite.cpp
index 9737807..06ab9d1 100644
--- a/src/nvtt/tests/testsuite.cpp
+++ b/src/nvtt/tests/testsuite.cpp
@@ -509,6 +509,9 @@ int main(int argc, char *argv[])
     nvtt::Context context;
     context.enableCudaAcceleration(!nocuda);
 
+    if (basePath.length() > 0) {
+        basePath.appendSeparator();
+    }
     basePath.append(set.basePath);
 
     FileSystem::changeDirectory(basePath.str());
-- 
2.28.0

